frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -63,6 +63,7 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
     const [validSize, setValidSize] = React.useState(true);
     const [restoreAccessMode, setRestoreAccessMode] = React.useState('');
     const [updatedProvisioner, setUpdatedProvisioner] = React.useState('');
+    const [isEncrypted, setEncrypted] = React.useState(false);
```

Comment:
```suggestion
    const isEncrypted = pvcSc?.parameters?.['encrypted'] === 'true';
```
This can be derived from pvcSC state value.